Sources/SourceKitD/SourceKitD.swift

```diff
@@ -70,11 +87,7 @@ extension SourceKitD {
   ///   - fileContents: The contents of the file that the request operates on. If sourcekitd crashes, the file contents
   ///     will be logged.
   public func send(_ req: SKDRequestDictionary, fileContents: String?) async throws -> SKDResponseDictionary {
-    logRequest(req)
-
-    let signposter = logger.makeSignposter()
-    let signpostID = signposter.makeSignpostID()
-    let signposterState = signposter.beginInterval("sourcekitd-request", id: signpostID, "Start")
```

Comment:
Did you decide the signpost wasn't useful? You could have `logRequest` return some opaque context that `log(response:)` and `log(crashedRequest:)` take if we want to keep it.

Reply:
Yeah. It was a mix of not useful and harder to implement. Just don’t make sense to design something with abstraction layers when it’s doubtful that it provides any real-world value.